src/components/candidates/candidate_list/candidates_list.tsx

```diff
@@ -0,0 +1,119 @@
+import React, { useMemo } from 'react';
+import {
+    useTable,
+    useGlobalFilter,
+    useFilters,
+    Column,
+} from 'react-table';
+import { useHistory } from 'react-router-dom';
+import './styles.css';
+import GlobalFilter from './globalFilter';
+import useAxios from 'axios-hooks';
+
+type TEducation = {
+    educational_institution: string,
+    faculty: string,
+    speciality: string
+};
+
+export type TCandidate = {
+    id: number,
+    first_name: string,
+    last_name: string,
+    phone: string,
+    email: string,
+    skype: string,
+    english_level: string,
+    country: string,
+    city: string,
+    primary_skills: string,
+    other_technologies: Array<string>,
```

Comment:
Better to use the same pattern for arrays in TS.
```suggestion
    other_technologies: string[],
```